dataactbroker/handlers/fileHandler.py

```diff
@@ -1607,6 +1607,14 @@ def add_list_submission_filters(query, filters):
             query = query.filter(Submission.submission_id.in_(sub_query))
         elif file_list:
             raise ResponseException("file_names filter must be null or an array", StatusCode.CLIENT_ERROR)
+    # User ID filter
+    if 'user_ids' in filters:
+        user_list = filters['user_ids']
+        if user_list and isinstance(user_list, list):
+            user_list = [int(user_id) for user_id in user_list]
```

Comment:
Should include a `try`-`except` here to catch non-integer values within the list.